README.md

```diff
@@ -11,6 +11,8 @@ cd initial_lengthening
 
 The first step consists in cloning the repositories of the DoReCo data, as well as CLTS. You can clone them with the following commands:
 
+TODO: Clone specific version/commit of the respective repository to highlight with which version of the sources this analysis was tested/done? I.e. a specific commit hash or tag of the repositories. (for CLTS: see comment below)
```

Comment:
Done for CLTS, remains to be done for DoReCo and my own code once the final code version is ready